### PR TITLE
.NET 6.0

### PR DESCRIPTION
**Pull request by erikzhang**
Related to #2625

### CODE REVIEW

File: src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs [hunk 1]
```diff
@@ -186,6 +186,8 @@ partial class ApplicationEngine
             [OpCode.MIN] = 1 << 3,
             [OpCode.MAX] = 1 << 3,
             [OpCode.WITHIN] = 1 << 3,
+            [OpCode.PACKMAP] = 1 << 11,
```
Comment (shargon) [inline src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs:189]: Maybe `PACKMAP` consume more resources

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ on:
   pull_request:
 
 env:
-  DOTNET_VERSION: 5.0.100
+  DOTNET_VERSION: 6.0.x
```
Comment (shargon) [inline .github/workflows/main.yml:9]: Deterministic version?

File: src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs (hunk 1 above)
Comment (erikzhang) [reply src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs:189]: Any one can test it?

File: .github/workflows/main.yml (hunk 2 above)
Comment (erikzhang) [reply .github/workflows/main.yml:9]: This is copied from the official template, I think there is no problem.

**Review by shargon — APPROVED** [review]

File: src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs (hunk 1 above)
Comment (shargon) [reply src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs:189]: @neo-project/ngd-shanghai ?

**Comment by superboyiii** [comment]
Why we have to change to .NET 6.0 so hurry?

**Review by vncoelho — APPROVED** [review]
Better to keep in the LTS

**Review by Liaojinghui — APPROVED** [review]

File: src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs (hunk 1 above)
Comment (Qiao-Jin) [reply src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs:189]: According to test it seems PACKMAP consume the same time as PACK

**Comment by erikzhang** [comment]
Merge?

**Review by shargon — APPROVED** [review]

**Pull request merged**